### PR TITLE
🧸  Updated rasusa

### PR DESCRIPTION
**Pull request by thanhleviet**
- Changed genome size input position

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `<SOFTWARE>.version.txt` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: software/rasusa/main.nf [hunk 1]
```diff
@@ -19,9 +19,8 @@ process RASUSA {
     }
 
     input:
-    tuple val(meta), path(reads)
+    tuple val(meta), path(reads), val(genome_size)
```
Comment (drpatelh) [inline software/rasusa/main.nf:22]: Is this to deal with the possibility of having 1 genome per sample?

Comment (thanhleviet) [reply software/rasusa/main.nf:22]: yes, in the bactmap pipeline, this process will receive samples genome size either predefined or estimated by sketch_mash

Comment (drpatelh) [reply software/rasusa/main.nf:22]: 🏋🏽

**Review by drpatelh — APPROVED** [review]

File: software/rasusa/main.nf (hunk 1 above)
Comment (aunderwo) [reply software/rasusa/main.nf:22]: @drpatelh in the meta.yml how do specify that the input should be a tuple 
`tuple val(meta), path(reads), val(genome_size)`
?

Comment (aunderwo) [reply software/rasusa/main.nf:22]: @drpatelh scratch that, I see that you'd examine the main.nf see what the items in the tuple are and then refer to the meta.yml to figure out the structure of those items

**Review by aunderwo — COMMENTED** [review]
Approved

**Pull request merged**